openapi/components/schemas/GatewayAccount.yaml

```
@@ -369,6 +369,29 @@ properties:
       do-nothing: |-
         Does nothing except return an approved `setup` transaction.
         This is the default behavior.
+  readyToPayoutInstruction:
+    type:
+      - 'string'
+      - 'null'
+    default: none
+    description: |-
+      Defines a mode of gateway payout behaviour for [StorefrontReadyToPayout](https://all-rebilly.redoc.ly/tag/Purchases/operation/StorefrontPostReadyToPayout).
+      If this value is `null`, payouts are not supported by the gateway.
+    enum:
+      - all
+      - covered-payout
+      - approved-payment
+      - none
+    x-enumDescriptions:
+      all: |-
+        Allow any amount to payout through gateway.
+      covered-payout: |-
+        Only payout amount of gateway what has been paid into gateway.
```

suggestion: to use the same text as the [UI copy](https://github.com/Rebilly/rebilly/pull/2354/files#diff-ca06702ad82a26622aa233ca38f7ac936dfa2e514eb1c8315cd29f6a8c224289)
```suggestion
        Payout from this gateway if it previously processed a payment for the same, 
        or a greater, amount.
```